spring-boot-project/spring-boot-test-autoconfigure/src/test/java/org/springframework/boot/test/autoconfigure/data/redis/DataRedisTestWithServiceConnectionBeanIntegrationTests.java

```diff
@@ -70,7 +70,7 @@ void testRepository() {
 	static class ContainerConfig {
 
 		@Bean
-		@ServiceConnection
+		@ServiceConnection(name = "redis")
```

Comment:
Should somewhere else be improved to avoid explicit `name`?

Reply:
I've created https://github.com/spring-projects/spring-boot/pull/36064 to handle this.